image-builder/image_builder.sh

```diff
@@ -34,6 +34,29 @@ readonly dax_header_sz=2
 # [2] - https://nvdimm.wiki.kernel.org/2mib_fs_dax
 readonly dax_alignment=2
 
+# The list of systemd units and files that are not needed in Kata Containers
+readonly -a systemd_units=(
+	"systemd-journald"
+	"systemd-journal-flush"
+	"systemd-journald-dev-log"
+	"systemd-udevd"
+	"systemd-udev-trigger"
+	"systemd-udevd-kernel"
+	"systemd-udevd-control"
+	"systemd-timesyncd"
+	"systemd-update-utmp"
+	"systemd-tmpfiles-setup"
+	"systemd-tmpfiles-cleanup"
+	"tmp"
```

Comment:
- Is that `tmp.mount`? Could you be specific.
- Nit: could you sort the list :)